man/faq/external-vector.Rmd

```diff
@@ -0,0 +1,68 @@
+
+```{r, child = "setup.Rmd", include = FALSE}
+```
+
+## Ambiguity between columns and external variables
+
+In selecting functions like `dplyr::select()` or `tidyr::pivot_longer()`,
```

Comment:
```suggestion
With selecting functions like `dplyr::select()` or `tidyr::pivot_longer()`,
```

Reply:
^^ this is English being weird in that this _could_ mean "In (you, the reader) selecting (as in choosing to use) functions like..."